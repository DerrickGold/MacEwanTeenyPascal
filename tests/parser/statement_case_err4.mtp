begin
	(*missing semicolon after case*)
	case (something) of
	first:
		s := 0
	second:
		s := 1
	else
		s := 99
	end
end
.