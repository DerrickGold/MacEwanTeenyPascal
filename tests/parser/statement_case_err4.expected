Syntax Error: line 6:
Unexpected token (TOK_ID, 'second'), likely missing a(n)  
  'TOK_KEY_ELSE' or
  'TOK_KEY_END' or
Possibly due to malformed or missing:
  Non-terminal: 'If Statement' or
  Non-terminal: 'While Statement' or
  Non-terminal: 'Write Statement' or
  Non-terminal: 'Read Statement' or
  Non-terminal: 'Case'
